core/src/Cuda/Kokkos_Cuda_BlockSize_Deduction.hpp

```diff
@@ -214,6 +213,7 @@ inline size_t get_shmem_per_sm_prefer_l1(cudaDeviceProp const& properties) {
     return 0;
   }() * 1024;
 }
+
```

Comment:
```suggestion
```

Reply:
I actually would like that to stay. I touched this function to delete the now wrong comment line, and added that space to visually separate the function end from the namespace closing.